lib/services/weed-db.service.ts

```
@@ -413,22 +414,23 @@ export async function ensureStrainExists(
-      (error.response.data.code === '23505' ||
-        (typeof error.response.data.message === 'string' &&
-          error.response.data.message.includes('duplicate key')))
+      typeof (error as any)?.response?.data !== 'undefined' &&
+      ((error as any).response.data.code === '23505' ||
+        (typeof (error as any).response.data.message === 'string' &&
+          (error as any).response.data.message.includes('duplicate key')))
     ) {
       // Try to fetch the existing strain by name
       const found = await fetchStrains({ search: strain.name });
       const match = found.data.find((s) => s.name.toLowerCase() === strain.name.toLowerCase());
       if (match && match.id) return String(match.id);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)